guava-tests/test/com/google/common/collect/ListsTest.java

```diff
@@ -629,6 +629,15 @@ public void testCartesianProduct_indexOf() {
     assertEquals(actual.indexOf(list(1, 1, 1)), -1);
   }
 
+  public void testCartesianProduct_lastIndexOf() {
+    List<List<Integer>> actual = Lists.cartesianProduct(list(1, 1), list(2, 3));
+    assertEquals(actual.lastIndexOf(list(1, 2)), 2);
```

Comment:
`assertEquals` actually expects its arguments in the other order -- expected, then actual.  Could you use Truth `assertThat(...).isEqualTo(...)` to avoid that confusion?